src/Compilers/CSharp/Portable/Binder/Binder_Deconstruct.cs

```diff
@@ -712,11 +712,49 @@ private BoundExpression BindDeconstructionDeclarationLocal(TypeSyntax typeSyntax
                     Error(diagnostics, ErrorCode.ERR_DeconstructionVarFormDisallowsSpecificType, designation);
```

Comment:
Do we have a test for this error in context of a global statement?

---
In reply to: [80101286](https://github.com/dotnet/roslyn/pull/13957#discussion_r80101286) [](ancestors = 80101286)